v3/src/components/graph/components/legend/legend.tsx

```diff
@@ -26,17 +25,16 @@ export const Legend = function Legend({
                                         legendAttrID, graphElt,
                                         onDropAttribute, onTreatAttributeAs, onRemoveAttribute
                                       }: ILegendProps) {
+  useMemo(() => legendAttrID ? [legendAttrID] : [], [legendAttrID])
```

Comment:
I think this can just be removed given that the function has no side effects and the return value isn't being used.